.babelrc

```diff
@@ -2,16 +2,18 @@
   "env": {
     "development": {
       "presets": ["es2015", "react"],
-      "plugins": ["react-hot-loader/babel"]
+      "plugins": ["react-hot-loader/babel", "transform-object-rest-spread"]
```

Comment:
Nice. It does seem a bit odd there isn't a "staging" key somewhere here, but I assume somewhere that definition would be moved to "development".